js/builder.js

```diff
@@ -142,15 +142,18 @@ function toggle_boost_tab(tab) {
     document.getElementById(tab + "-boost-tab").classList.add("selected-btn");
 }
 
-let tabs = ['overall-stats', 'offensive-stats', 'defensive-stats'];
-function show_tab(tab) {
+// let tabs = ['overall-stats', 'offensive-stats', 'defensive-stats'];
+
+function show_tab(target, tabs) {
+    //console.log(itemFilters)
+
```

Comment:
Is this the only one rn? can `toggle_boost_tab` above also be merged into this?

also can you document a bit exactly what the behavior is? it seems to depend on the ids' formatting being a specific way

Reply:
done